### PR TITLE
Add `all_whl_requirements` to match `all_requirements`

### PR DESCRIPTION
**Pull request by person142**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Does not include precompiled binaries, eg. `.par` files. See [CONTRIBUTING.md](/CONTRIBUTING.md) for info
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
There isn't an equivalent to `all_requirements` for `whl_requirement`.

## What is the new behavior?
A list `all_whl_requirements` is added to `requirements.bzl` which contains all of the wheel targets.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
The `all_whl_requirements` list is handy if you want to write a target that takes all of the wheels built by `rules_python` and upload them to a private wheelhouse say.

### CODE REVIEW

**Comment by person142** [comment]
@googlebot I signed it!

File: python/pip_install/extract_wheels/lib/bazel.py [hunk 1]
```diff
@@ -82,10 +87,14 @@ def requirement(name):
            name_key = name.replace("-", "_").replace(".", "_").lower()
            return "{repo}//pypi__" + name_key
 
+        all_whl_requirements = [{whl_requirement_labels}]
```
Comment (hrfuller) [inline python/pip_install/extract_wheels/lib/bazel.py:90]: Nit: Put this underneath all_requirements so the macros are separated visually from the static definitions.

**Review by hrfuller — APPROVED** [review]
@thundergolfer This LGTM.

File: python/pip_install/extract_wheels/lib/bazel.py (hunk 1 above)
Comment (person142) [reply python/pip_install/extract_wheels/lib/bazel.py]: Done in https://github.com/bazelbuild/rules_python/pull/377/commits/b6886f26b7598ebe1e880230f36500b4c0f5a7b4.

**Review by hrfuller — APPROVED** [review]

**Review by thundergolfer — APPROVED** [review]

**Pull request merged**